src/cpp/benders/adequacy_criterion/AdequacyCriterion.cpp

```diff
@@ -47,27 +46,27 @@ void OuterLoop::Run() {
   // last prints
   PrintLog();
   benders_->mathLoggerDriver_->Print(benders_->GetCurrentIterationData());
-  benders_->SaveOuterLoopSolutionInOutputFile();
+  benders_->SaveAdequacyCriterionSolutionInOutputFile();
   // TODO general-case
   //  cuts_manager_->Save(benders_->AllCuts());
   benders_->free();
 }
 
-void OuterLoop::PrintLog() {
+void AdequacyCriterion::PrintLog() {
   std::ostringstream msg;
   auto logger = benders_->_logger;
   logger->PrintIterationSeparatorBegin();
-  msg << "*** Outer loop: " << benders_->GetBendersRunNumber();
+  msg << "*** Adequacy Criterion: " << benders_->GetBendersRunNumber();
```

Comment:
`Adequacy criterion loop`  maybe ? As we print the number of iterations of a loop here

Reply:
updated